OpenRA.Mods.Common/Traits/SelfHealing.cs

```diff
@@ -18,7 +18,8 @@ class SelfHealingInfo : UpgradableTraitInfo, Requires<HealthInfo>
 	{
 		public readonly int Step = 5;
 		public readonly int Delay = 5;
-		public readonly float HealIfBelow = .5f;
+		[Desc("Heal if current health is below this percentage of full health.")]
+		public readonly int HealIfBelow = 50;
```

Comment:
Do we want to rename these fields to include `Percent` in their name?  (I know we do this for some fields, but I'm not sure how widely that convention is used).